kingpin/actors/aws/cloudformation.py

```diff
@@ -762,6 +781,9 @@ class Stack(CloudFormationBaseActor):
         'timeout_in_minutes': (int, 60,
                                'The amount of time that can pass before the '
                                'stack status becomes CREATE_FAILED'),
+        'enable_termination_protection': (TerminationProtectionConfig, None,
```

Comment:
Add to the docstrings above fro this option - also while you're there, can you include the `role_arn` docs too?

Reply:
Ah, missed those particular ones.